exercises/go-1-a/go_1_a_test.go

```diff
@@ -37,6 +37,6 @@ func TestSixtyFive(t *testing.T) {
 func AssertDescribe(t *testing.T, input int, expected string) {
 	actual := Describe(input)
 	if actual != expected {
-		t.Errorf("Describe(%d) = %q, expected %q.", input, actual, expected)
+		t.Errorf("Describe(%d)\n   got: %q\n  want: %q.", input, actual, expected)
```

Comment:
This is nice! We should probably also do this for the other Go test files at some point.

Reply:
I can handle that.